relocation.bs

```diff
@@ -733,6 +733,23 @@ void do_something_03(std::mutex& m)
 }
 ```
 
+### Relocation elision ### {#reloc-elision}
```

Comment:
This section is a nice addition, but I would move it before "Discarded reloc expressions".